docs/openapi.yaml

```diff
@@ -576,6 +576,70 @@ paths:
         502:
           $ref: '#/components/responses/502'
   /serial-publication:
+    get:
+      x-amazon-apigateway-integration:
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${SearchSerialPublicationByQueryFunction.Arn}/invocations
+        httpMethod: POST
+        type: "AWS_PROXY"
+      tags:
+        - SerialPublication
+      summary: Search for series/journal by name, issn, or year
+      description: Returns a list of Series and/or Journals
+      operationId: SearchSerialPublicationByQuery
+      parameters:
+        - in: query
+          name: year
+          schema:
+            type: integer
```

Comment:
There are more restrictions, no?

Reply:
Not after https://github.com/BIBSYSDEV/nva-publication-channels-java/pull/61